uniflow/flow/config.py

```diff
@@ -134,6 +134,70 @@ class TransformCopyConfig(TransformConfig):
     )
     model_config: ModelConfig = field(default_factory=lambda: {})
 
+class TransformForClassificationOpenAIGPT3p5Config(TransformConfig):
```

Comment:
please rename this Config because this is not a `classification` config but a `generation` config.